projects/go-lib/src/lib/components/go-action-sheet/go-panel/go-panel.component.html

```diff
@@ -1,6 +1,7 @@
 <!-- External Link -->
 <a
   class="go-panel"
+  [disabled]="disablePanel" 
```

Comment:
So this won't work because `disabled` isn't a property on an `a` element. What do you think about adding a modifier class for the `go-panel` class instead?

```
.go-panel {
  &--disabled {
    opacity: 0.5;
  }
}